src/types/common.d.ts

```diff
@@ -0,0 +1,62 @@
+/** Element attributes. */
+export interface ElementAttributes {
```

Comment:
Sorry, I didn't notice this in the last change, but let's keep `ElementAttributes` and `ElementStyle` in `element.ts` please.

`RenderContext` is okay here, but I'd like to reorganize it later.

Reply:
This comes in the 2nd PR with the comments. If I do the change here we lose the diff. I had not forgotten it :)

Reply:
Great, thanks!